README.md

```diff
@@ -33,3 +33,35 @@ The repository also contains some tools, including the map editor and various
 GM tools.
 
 The tools live in the /tools folder.
+
+## Running Locally
```

Comment:
This is a big step forward and very helpful to me.  Not having to go to the deployed server for development makes testing go much faster.  I think we might consider making a little script that creates and populates a "standard" datastore that contains some suitably real/complex game states that can be used for testing/simulation (maybe just copying from the actual games that have already run).  That could be its own issue to tackle separately though.

Reply:
Agreed. I was planning on writing such a tool (it is easy) but just haven't had time.